src/app/shared/dialogs/dialogs-prompt.component.html

```
@@ -41,14 +41,25 @@
 }
-<p [innerHTML]=data.extraMessage></p>
-<b class = break-word>{{data.displayName | truncateText:140}}</b>
-@if (data.displayDates) {
-  <b>{{data.displayDates.startDate | date : 'mediumDate' : isDateUtc ? '+0000' : undefined }} - {{data.displayDates.endDate | date: 'mediumDate' : isDateUtc ? '+0000' : undefined }}</b>
-}
-@for (label of labels; track label; let last = $last) {
-  <p><b><planet-label [label]="label.field"></planet-label>: {{label.value}}</b></p>
-}
+  <b class="break-word">{{data.displayName | truncateText:140}}</b>
+  @if (data.rules) {
+    <div style="margin-top: 12px;">
```

HTMLHint (1.9.2)

[warning] Inline style [  style="margin-top: 12px;" ] cannot be used.

(inline-style-disabled)